README.md

```diff
@@ -75,3 +73,11 @@ To begin testing your scenarios
 - Subject line, use the imperative form of a verb
   - GOOD "add contributing guidelines"
   - BAD "adding contribuing guidelines"
+
+## Contributing
```

Comment:
There's already a contributing heading on line 33 @vindard

Reply:
Moved the forking steps up to be under that existing header.